docs/onnx.md

```diff
@@ -1,10 +1,442 @@
-# PyTorch ONNX utils
-ONNX utilties to implement features in onnx-chainer
+# pytorch_pfn_extras.onnx
 
-## Requirements
-Install onnx package as an additional requirements to use this package
+Extensions of `torch.onnx.export` for PFN tools.
```

Comment:
I believe these utils are also convenient for some external users testing ONNX runtimes 😉 

```suggestion
Extensions to `torch.onnx.export`.
```

Reply:
SGTM :+1: